core/src/main/java/org/bitcoinj/manager/ManagerFiles.java

```diff
@@ -119,7 +119,7 @@ public void saveNow() throws IOException {
         // but they will serialize (using different temp files).
         if (executor.isShutdown())
             return;
-        log.info("Saving manager file: {}", file.toPath());
+        log.info("Saving manager file: {}", file.getAbsolutePath());
```

Comment:
`toPath()` is not available on Android 6 or 7.  `getAbsolutePath()` is used in many other places in DashJ.

Reply:
Perhaps we should have been building DashJ for Java 7 to avoid some of these types of errors.